src/App.tsx

```diff
@@ -1,223 +1,109 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React, { useContext, useEffect, useRef } from 'react';
+import React, {
+  useContext,
+  useEffect,
+  useRef,
+  useState,
+} from 'react';
+import classNames from 'classnames';
 import { AuthContext } from './components/Auth/AuthContext';
+import { getTodos } from './api/todos';
+import { Notification } from './components/Notifications';
+import { NewTodo } from './components/NewTodo';
+import { FilterTodo } from './components/Filter';
+import { TodoList } from './components/TodoList';
+import { Filter } from './types/Filter';
+import { Notifications } from './types/Notifications';
+import { Todo } from './types/Todo';
 
 export const App: React.FC = () => {
   // eslint-disable-next-line @typescript-eslint/no-unused-vars
   const user = useContext(AuthContext);
```

Comment:
You can use this state in dependencies array in the useEffect below, so you can delete this comment